packages/ckeditor5-engine/src/view/view.ts

```diff
@@ -810,6 +810,48 @@ export default class View extends /* #__PURE__ */ ObservableMixin() {
 	}
 }
 
+/**
+ * Get the view range from the given DOM event.
+ *
+ * @param view The view instance.
+ * @param domEvent The DOM event.
+ * @returns The view range or `null` if the range cannot be created.
+ */
+export function getPointViewRange(
```

Comment:
The `view.ts` does not look like a perfect spot for such a helper function especially since it is not used here. I'd suggest extracting this helper to the `utils` package as other DOM helpers, for example this one:

https://github.com/ckeditor/ckeditor5/blob/08270aafcc6e7e4fb4f4a4d2ebbc7451f031d303/packages/ckeditor5-utils/src/dom/scroll.ts#L60

Note that currently this helper expects a `View` instance as a parameter, but it is used only once so we could extract DOM-focused behavior to a helper util and use DOM to view mapping outside the helper itself.